keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/ClusterHandler.kt

```diff
@@ -337,8 +346,7 @@ class ClusterHandler(
     val spec = ClusterSpec(
       moniker = exportable.moniker,
       imageProvider = if (base.buildInfo?.packageName != null) {
-        ArtifactImageProvider(
-          deliveryArtifact = DebianArtifact(name = base.buildInfo.packageName!!))
+        ReferenceArtifactImageProvider(reference = base.buildInfo.packageName.toString())
```

Comment:
This seems unrelated?

Reply:
oh it totally is. But I wanted to do it. I can back it out if you care